platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/StsRealm.java

```diff
@@ -533,6 +534,10 @@ public void setUsernameAttributeList(List<String> usernameAttributeList) {
     this.usernameAttributeList = usernameAttributeList;
   }
 
+  public void setUsernameAttributeList(String usernameAttributeList) {
+    this.usernameAttributeList = Arrays.asList(usernameAttributeList.split("\\s*,\\s*"));
```

Comment:
Had to add because we get a comma-separated string from blueprint and metatype. The alternative is to change both of those to lists.